components/host-dashboard/HostInfoCard.js

```diff
@@ -163,6 +168,37 @@ const HostInfoCard = ({ host }) => {
           <ConnectTransferwiseButton isConnected={Boolean(host.transferwiseBalances)} />
         </Container>
       </Flex>
+      {host.stripeIssuingBalance && (
+        <React.Fragment>
+          <Separator />
+          <Flex flexDirection="column" justifyContent="space-between" flex="1 1 33%">
```

Comment:
Perhaps we should change the `justifyContent` to `flex-start` and use add `margin-top` so the amounts are all vertically aligned?

![image](https://github.com/opencollective/opencollective-frontend/assets/2119706/2f9f68dc-cd7a-4857-9af4-749e0006c65d)